examples/3dtiles.html

```diff
@@ -98,7 +98,10 @@
             itowns.View.prototype.addLayer.call(globe, $3dTilesLayerRequestVolume);
 
             // Add the UI Debug
-            new debug.Debug(globe, viewerDiv);
+            const d = new debug.Debug(globe, menuGlobe.gui);
+            new debug.TileDebug(menuGlobe.gui, globe, globe.wgs84TileLayer, d);
```

Comment:
If we don't need the created object, and as it seems it never uses `this`, let's expose regular function `createTileDebugUi` instead of constructors.